test/elastic/integration/index_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule Elastic.Integration.IndexTest do
+  use Elastic.IntegrationTestCase
+  alias Elastic.{HTTP, Index}
+
+  @tag integration: true
+  test "close/1 & open/1" do
+    Answer.index(1, %{text: "Hello world!"})
+
+    cat_result = get_index_response(HTTP.get("_cat/indices?format=json"), Index.name("answer"))
+    assert %{"status" => "open"} = cat_result
+
+    Index.close("answer")
+
+    cat_result = get_index_response(HTTP.get("_cat/indices?format=json"), Index.name("answer"))
+    assert %{"status" => "close"} = cat_result
+
+    Index.open("answer")
+
+    cat_result = get_index_response(HTTP.get("_cat/indices?format=json"), Index.name("answer"))
+    assert %{"status" => "open"} = cat_result
+  end
+
+  defp get_index_response({:ok, 200, indices}, name) do
```

Comment:
I thought about cover more of index related functions in same test in the future, so I think for now we can keep it, maybe later we can find better way.

Also there are direct calls to `_cat`. Do you see it useful to have `Cat` module?